coding_conventions.md

```diff
@@ -239,6 +240,29 @@ const ChargeModuleTokenService = require('../../app/services/charge-module-token
 // Start testing!
 ```
 
+## Unit test descriptive text
+
+When writing the descriptive text in either the `describe()` or `it()` blocks of a unit test, we follow the convention of wrapping the text in single quotes.
+
+If there is a word(s) within this text string that we want to emphasise then we follow the convention of wrapping the word(s) in double quotes.
+
+```javascript
+// Good
+it('returns a "true" success status', async () => {
+  expect(succeeded).to.be.true()
+})
+
+// Bad
+it('returns a `true` success status', async () => {
+  expect(succeeded).to.be.true()
+})
+
+// Also Bad
+it("returns a 'true' success status", async () => {
+  expect(succeeded).to.be.true()
+})
```

Comment:
Could we add the example I most commonly see which I was trying to encourage folks to avoid using.

```suggestion
// Bad
it('returns a \'true\' success status', async () => {
  expect(succeeded).to.be.true()
})

// Also bad
it('returns a `true` success status', async () => {
  expect(succeeded).to.be.true()
})

// Also bad
it("returns a 'true' success status", async () => {
  expect(succeeded).to.be.true()
})
```

Reply:
Go on then :grin: